Documentation/UsingSetting/_Sets/_TcaOverridesSystemplateV12.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use TYPO3\CMS\Core\Information\Typo3Version;
+use TYPO3\CMS\Core\Utility\ExtensionManagementUtility;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+defined('TYPO3') or die();
+
+call_user_func(function()
+{
+    $extensionKey = 'my-extension';
+    $versionInformation = GeneralUtility::makeInstance(Typo3Version::class);
+    if ($versionInformation->getMajorVersion() < 13) {
```

Comment:
This is problematic, when you provide a 3rd-party extension which is compatible with TYPO3 v12 and v13. You force people with this to use site sets. Without site sets the extension is not usable anymore (or you have to define that TypoScript in your site package manually).